### ISSUE TITLE
Uncaught TypeError: Cannot read property 'findKeyBindings' of undefined

### ISSUE THREAD

**Issue by philippeowagner** [mark-hahn/command-toolbar#9]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.206.0
**System**: Mac OS X 10.10.3
**Thrown From**: [command-toolbar](https://github.com/mark-hahn/command-toolbar) package, v1.0.11

### Stack Trace

Uncaught TypeError: Cannot read property 'findKeyBindings' of undefined

```
At /Users/phi/.atom/packages/command-toolbar/lib/finder.coffee:28

TypeError: Cannot read property 'findKeyBindings' of undefined
  at CommandPaletteView.module.exports.CommandPaletteView.attach (/Users/phi/.atom/packages/command-toolbar/lib/finder.coffee:28:31)
  at HTMLDivElement.<anonymous> (/Users/phi/.atom/packages/command-toolbar/lib/toolbar-view.coffee:31:22)
  at HTMLDivElement.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLDivElement.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -4:46.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -3:35.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:49.5.0 core:paste (atom-text-editor.editor.mini.is-focused)
  3x -2:48.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -2:41.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -2:16.9.0 core:paste (atom-text-editor.editor.mini.is-focused)
     -2:15.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  5x -2:12.8.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -2:10.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:09.0 core:backspace (atom-text-editor.editor.mini.is-focused)
  2x -2:08.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:09.0 snippets:next-tab-stop (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -0:09.0 snippets:expand (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -0:09.0 editor:indent (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -0:08.1.0 core:move-left (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -0:06.9.0 core:copy (atom-text-editor.editor.vim-mode.command-mode.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
clipboard-plus, v0.1.0
color-picker, v2.0.4
command-toolbar, v1.0.11
custom-title, v1.0.1
linter, v0.12.6
vim-mode, v0.51.0

# Dev
No dev packages
```

**Comment by fkj** [comment]
I am also experiencing this issue. The error is thrown whenever I click the "hamburger" button.

**Atom Version**: 0.209.0
**System**: Microsoft Windows 8.1
**Thrown From**: [command-toolbar](https://github.com/mark-hahn/command-toolbar) package, v1.0.11


### Stack Trace

Uncaught TypeError: Cannot read property 'findKeyBindings' of undefined

```
At C:\Users\fkjac_000\.atom\packages\command-toolbar\lib\finder.coffee:28

TypeError: Cannot read property 'findKeyBindings' of undefined
  at CommandPaletteView.module.exports.CommandPaletteView.attach (C:\Users\fkjac_000\.atom\packages\command-toolbar\lib\finder.coffee:28:31)
  at HTMLDivElement.<anonymous> (C:\Users\fkjac_000\.atom\packages\command-toolbar\lib\toolbar-view.coffee:31:22)
  at HTMLDivElement.jQuery.event.dispatch (C:\Users\fkjac_000\AppData\Local\atom\app-0.209.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4435:9)
  at HTMLDivElement.elemData.handle (C:\Users\fkjac_000\AppData\Local\atom\app-0.209.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4121:28)

```

### Commands

```
  2x -0:50.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:49.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      ".exe"
    ]
  }
}
```

### Installed Packages

```coffee
# User
build, v0.37.0
command-toolbar, v1.0.11
language-d, v3.2.1

# Dev
No dev packages
```

**Comment by lygstate** [comment]
+1 for this

**Comment by mark-hahn** [comment]
Thanks to everyone.  Fixed in 1.0.12.
